src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -91,7 +91,10 @@ export function createMessageContent(
         msgtype: isEmote ? "m.emote" : "m.text",
         body: body,
     };
-    const formattedBody = htmlSerializeIfNeeded(model, { forceHTML: !!replyToEvent });
+    const formattedBody = htmlSerializeIfNeeded(model, {
+        forceHTML: !!replyToEvent,
```

Comment:
I don't think this is still needed. 
We had to force HTML rendering when replying because of the reply fallbacks, but this has been scrapped as part of [MSC2781](https://github.com/matrix-org/matrix-doc/pull/2781).

Let me confirm with someone from the web team